index.html

```diff
@@ -0,0 +1,650 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="Assets/reset.css">
+  <link rel="stylesheet" href="Assets/style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/fc8cc3c5c9.js"></script>
+  <title>The Next Web</title>
+</head>
+
+<body>
+  <header>
+    <div class="nav-wrapper">
+      <div class="top-nav">
+        <ul class="top-nav-left">
+          <li class="tn-list-item"><a class="top-nav-link" href="#">News</a></li>
+          <li class="tn-list-item"><a class="top-nav-link" href="#">Events</a></li>
+          <li class="tn-list-item"><a class="top-nav-link" href="#">Bussiness</a></li>
+          <li class="tn-list-item"><a class="top-nav-link" href="#">AMAs</a></li>
+          <li class="tn-list-item"><a class="top-nav-link" href="#">Spaces</a></li>
+          <li class="tn-list-item"><a class="top-nav-link" href="#">Terms and Conditions</a></li>
+        </ul>
+        <ul class="top-nav-right">
+          <li class="tn-list-item"><a class="top-nav-link" href="#">About</a></li>
+          <li class="tn-list-item"><a class="top-nav-link" href="#">Advertise</a></li>
+          <li class="tn-list-item"><a class="top-nav-link" href="#">Jobs</a></li>
+          <li class="tn-list-item"><a class="top-nav-link" href="#">Contact</a></li>
+        </ul>
+      </div>
+      <div class="main-nav">
+        <a class="dark-logo-wrapper" href="#"><img class="dark-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="Logo"></a>
+        <ul class="main-nav-left">
+          <li class="mnl-list-item"><a class="mnl-link" href="#">LATEST</a></li>
+          <li class="mnl-list-item"><a class="mnl-link" href="#">HARD WORK</a></li>
+          <li class="mnl-list-item"><a class="mnl-link" href="#">PLUGGED</a></li>
+          <li class="mnl-list-item"><a class="mnl-link" href="#">README</a></li>
+          <li class="mnl-list-item"><a class="mnl-link" href="#">GROWTH HEADQUATERS</a></li>
+        </ul>
+        <ul class="main-nav-right">
+          <li class="mnr-list-item"><a class="mnr-link" href="#"><i class="fas fa-search"></i></a></li>
+          <li class="mnr-list-item"><a class="mnr-link" href="#"><i class="fas fa-bars"></i></a></li>
+          <li class="mnr-list-item"><a class="mnr-link" href="#"><img class="logo" src="images/logo.png" alt="logo" style="line-height:1"></a></li>
+          <li class="mnr-list-item"><a href="#" class="counter">00:20:30:20</a></li>
```

Comment:
Please do not use `inline styling`, you can move the style into the CSS file and improve your code's readability